setup.cfg

```diff
@@ -1,7 +1,3 @@
-[bdist_wheel]
-# The code is written to work on both Python 2 and Python 3.
-universal=1
-
 [bdist_rpm]
 # Due to changes in Python's ConfigParser, this only works in Python 3. For Python 2, remove one %
```

Comment:
Tangentially related: does this comment have any value anymore?

Reply:
probably not